packages/auth/src/grant/routes.ts

```diff
@@ -1,26 +1,31 @@
+import * as crypto from 'crypto'
+import { URL } from 'url'
 import { AppContext } from '../app'
 import { GrantService, GrantRequest } from './service'
 import { ClientService } from '../client/service'
 import { BaseService } from '../shared/baseService'
 import { isAccessRequest } from '../access/types'
+import { IAppConfig } from '../config/app'
 
 interface ServiceDependencies extends BaseService {
   grantService: GrantService
   clientService: ClientService
+  config: IAppConfig
 }
 
 export interface GrantRoutes {
   create(ctx: AppContext): Promise<void>
-  // interaction: {
-  //   get(ctx: AppContext): Promise<void>
-  //   post(ctx: AppContext): Promise<void>
-  // }
+  interaction: {
+    get(ctx: AppContext): Promise<void>
+    post(ctx: AppContext): Promise<void>
```

Comment:
I think `start` and `finish` would be great. With the way I've structured the grant routes service these endpoints would be structured behind `grantRoutes.interaction.start/finish` so that should efficiently communicate what it is.